### PR TITLE
[JENKINS-35178] Fix close button in Safari

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-35178](https://issues.jenkins-ci.org/browse/JENKINS-35178)

@reviewbybees
@jenkinsci/code-reviewers 
### Screenshot on Safari

jenkins-35178

Dedicated to @oleg-nenashev 😄

### CODE REVIEW

File: war/src/main/js/widgets/config/tabbar.less [hunk 1]
```diff
@@ -299,42 +299,42 @@
         position:relative;
       }  
       .repeatable-delete {
-        position:absolute;
-        top:-2px;
-        right:35px;
-        height:18px;
-        width:32px;
-        .border-radius-bottom(3px);
```
Comment (kzantow) [inline war/src/main/js/widgets/config/tabbar.less]: removed?

File: war/src/main/js/widgets/config/tabbar.less [hunk 2]
```diff
@@ -299,42 +299,42 @@
...
           font-weight: bold;
-          text-indent: 0;
-          display: block;
           position: absolute;
-          left: 0;
-          right: 0;
-          top: 0;
-          bottom: 0;
+          display: block;
+          left: 0px;
```
Comment (kzantow) [inline war/src/main/js/widgets/config/tabbar.less]: :( css people tend to like these left as just `0`, with no units

File: war/src/main/js/widgets/config/tabbar.less (hunk 1 above)
Comment (recena) [inline war/src/main/js/widgets/config/tabbar.less]: Inherited through `<button />`

File: war/src/main/js/widgets/config/tabbar.less [hunk 3]
```diff
@@ -299,42 +299,42 @@
...
+          top: 0px;
+          width: 30px;
           text-align: center;
           font-size: 12px;
-          line-height: 18px;
```
Comment (kzantow) [inline war/src/main/js/widgets/config/tabbar.less]: also removed? this will affect vertical positioning

Comment (recena) [inline war/src/main/js/widgets/config/tabbar.less]: @oleg-nenashev No needed with this approach.

**Comment by recena** [comment]
@kzantow your comment was addressed [here](https://github.com/jenkinsci/jenkins/pull/2378/commits/d533954e26a5bcc7b07d341358697165551a6223).

File: war/src/main/js/widgets/config/tabbar.less (hunk 1 above)
Comment (kzantow) [inline war/src/main/js/widgets/config/tabbar.less]: No, it inherited a border-radius of 3px, it looks like. I think this was overriding it with a _bottom_ only a 3px border radius.

File: war/src/main/js/widgets/config/tabbar.less [hunk 4]
```diff
@@ -299,42 +299,42 @@
...
+          padding: 0 !important;
+          margin: 0 !important;
           background-color: @danger;
           color: @light-backgrond;
           border: 1px solid @danger-line;
         }
-        button:focus,
-        button:hover{
+
+        button:focus, button:hover {
```
Comment (kzantow) [inline war/src/main/js/widgets/config/tabbar.less]: :ant: separate lines is a better approach

File: war/src/main/js/widgets/config/tabbar.less (hunk 2 above)
Comment (recena) [inline war/src/main/js/widgets/config/tabbar.less]: I totally agree. My mistake.

File: war/src/main/js/widgets/config/tabbar.less (hunk 1 above)
Comment (recena) [inline war/src/main/js/widgets/config/tabbar.less]: Sure, addressed [here](https://github.com/jenkinsci/jenkins/pull/2378/commits/0b9aa535c24946ebd20b423447e844236e340a54)

**Comment by recena** [comment]
@kzantow This is the visual result:

jenkins-35178-2

**Comment by aheritier** [comment]
:bee:

**Comment by recena** [comment]
@reviewbybees done

**Comment by oleg-nenashev** [comment]
🐝 and 👍 if it has been tested in standard browsers

**Comment by recena** [comment]
@oleg-nenashev Thanks!

**Comment by recena** [comment]
@oleg-nenashev I did tests using Firefox, Safari and Chrome on Mac. More tests are always welcome.

**Pull request merged**